### PR TITLE
v8.0.0-alpha.0

### PR DESCRIPTION
**Pull request by MBilalShafi**

### CODE REVIEW

**Review by flaviendelangle — COMMENTED** [review]
@DanailH and @joserodolfofreitas were proposing not to release it this week
I would allow a small anouncement effect
No strong opinion on my side

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,95 @@
...
+Following are all team members who have contributed to this release:
+@cherniavskii, @flaviendelangle, @LukasTy, @MBilalShafi, @michelengelen, @samuelsycamore, @noraleonte, @romgrk.
+
+<!--/ HIGHLIGHT_ABOVE_SEPARATOR /-->
+
+### Data Grid
+
+#### `@mui/x-data-grid@8.0.0-alpha.0`
+
+- [DataGrid] Fix `GridPanelAnchor` positioning (#15022) @k-rajat19
```
Comment (flaviendelangle) [inline CHANGELOG.md:29]: Not sure if those should be here since they are already in the v7 release that is before this one

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,95 @@
...
+
+#### `@mui/x-data-grid-premium@8.0.0-alpha.0` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+Same changes as in `@mui/x-data-grid-pro@8.0.0-alpha.0`.
+
+### Date and Time Pickers
+
+#### `@mui/x-date-pickers@8.0.0-alpha.0`
+
+- [fields] Enable the new field DOM structure by default (#14651) @flaviendelangle
```
Comment (flaviendelangle) [inline CHANGELOG.md:53]: I need to write a highlight if we do release it

**Comment by MBilalShafi** [comment]
> @DanailH and @joserodolfofreitas were proposing not to release it this week

I'm OK with postponing till the next week.

**Comment by LukasTy** [comment]
> I'm OK with postponing till the next week.

I also vote for postponing at least until the start of November so that we get a bit more stuff into the first release. 🤔 👍

**Comment by MBilalShafi** [comment]
Will reopen in subsequent weeks when more changes are stacked up.

**Pull request closed without merging**